tests/test_splitter.py

```diff
@@ -8,8 +8,59 @@
 
 class TestSplitter(TestCase):
```

Comment:
I'm confused, and this is probably just due to my lack of understanding of whats going on, what exactly is this testing? It looks like its just running the `splitter` program and making sure it file extension is correct. Is there any testing to make sure the data is *actually split*? Also should these tests clean up afterwards and remove the files created by the `splitter` during the testing? Otherwise, how can you tell that the tests really worked? They might just be checking files from previous tests. Unless I'm misunderstanding something, it looks like both tests call `splitter` in the same folder which means that `splitter` creates the same output files with the same name in the same folder.